src/store/modules/product/actions.ts

```diff
@@ -40,6 +42,37 @@ const actions: ActionTree<ProductState, RootState> = {
     return resp;
   },
 
+  async findProduct ({ commit, state }, payload) {
+    // Show loader only when new query and not the infinite scroll
+    if (payload.viewIndex === 0) emitter.emit("presentLoader");
+    let resp;
+    try {
+      resp = await ProductService.fetchProducts({
+        // used sku as we are currently only using sku to search for the product
+        "filters": ['sku: ' + payload.queryString],
```

Comment:
This will only search on sku. Pass it as keyword in parameters, so that query can be performed on multiple fields (upc productId sku groupId productName productCategoryNames) for getting results

Reply:
updated sir